packages/renderer/src/lib/preferences/PreferencesConnectionActions.svelte

```diff
@@ -184,15 +184,16 @@ function getLoggerHandler(
             leftPosition="left-[0.12rem]" />
         {/if}
         {#if connection.lifecycleMethods.includes('delete')}
-          <div class="mr-2 text-sm">
-            <LoadingIconButton
-              clickAction={() => deleteConnectionProvider(provider, connection)}
-              action="delete"
-              icon={faTrash}
-              state={connectionStatus}
-              leftPosition="left-[0.15rem]" />
-          </div>
+          <LoadingIconButton
+            clickAction={() => deleteConnectionProvider(provider, connection)}
+            action="delete"
+            icon={faTrash}
+            state={connectionStatus}
+            leftPosition="left-[0.15rem]" />
```

Comment:
question: why do we have this change ?

AFAIK we're only adding advanced actions

Reply:
Current (in this PR): 
![image](https://github.com/user-attachments/assets/ad4a89ab-08bc-4b6d-bb25-79b3d8766906)

Original (without the div change):
![image](https://github.com/user-attachments/assets/6227febf-0981-4213-8cb8-79de9c49464c)
- there is s big space between items